ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1627,3 +1627,17 @@ def igesin(self, fname="", ext="", **kwargs):
             out = super().igesin(basename, **kwargs)
 
         return out
+
+    @wraps(_MapdlCore.cmatrix)
+    def cmatrix(self, symfac="", condname="", numcond="", grndkey="",
+                capname="", **kwargs):
+        """Wrap the CMATRIX command to update correctly the `last_response` command. """ 
+
+        """The CMATRIX command needs to run in non-interactive mode"""
+        empty_output = super().cmatrix(self, symfac, condname, numcond, grndkey, capname, **kwargs)
```

Comment:
```suggestion
        super().cmatrix(self, symfac, condname, numcond, grndkey, capname, **kwargs)
```
No reason to store this output.

Reply:
Ah, turns out we're not testing this.

Add test.

Reply:
I feel like this deserve a whole new issue. So I'm going to open it. Feel free to close this suggestion or leave it open until we are done with the new issue. 

See new issue #588

Reply:
See comments below regarding adding testing with features.